cluster/addons/podsecuritypolicies-rbac/gce/kubelet-binding.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
a comment in this indicating that this is intended to be used in combination with the `Node` authorization mode and `NodeRestriction` admission plugin, so that kubelets are only allowed to create mirror pods bound to themselves